src/routes/landing/components/CityEntry.js

```diff
@@ -31,11 +34,31 @@ type PropsType = {|
 class CityEntry extends React.PureComponent<PropsType> {
   render () {
     const { city, language, filterText } = this.props
+    const getMunicipality = (city, filterText): Array<CityModel> => {
```

Comment:
Either move this out of `render` and create a separate method (maybe even a `renderMunicipalities` or something which includes `{
          municipalities.map(municipality => (
            <MunicipalityItem key={municipality} search={filterText}>
              {municipality}
            </MunicipalityItem>
          ))
        }`) or inline it later at that occurance.